src/main/java/com/avaje/ebean/dbmigration/ddlgeneration/platform/MsSqlServerHistoryDdl.java

```diff
@@ -0,0 +1,85 @@
+package com.avaje.ebean.dbmigration.ddlgeneration.platform;
+
+import com.avaje.ebean.dbmigration.ddlgeneration.DdlBuffer;
+import com.avaje.ebean.dbmigration.ddlgeneration.DdlWrite;
+import com.avaje.ebean.dbmigration.model.MTable;
+
+import java.io.IOException;
+
+/**
+ * MySql history support using DB triggers to maintain a history table.
+ */
+public class MsSqlServerHistoryDdl extends DbTriggerBasedHistoryDdl {
+
+
+  public MsSqlServerHistoryDdl() {
+    this.currentTimestamp = "CURRENT_TIMESTAMP";
+    this.sysPeriodType = "datetime";
+  }
+
+  @Override
+  protected void dropTriggers(DdlBuffer buffer, String baseTable) throws IOException {
+
+    buffer.append("drop trigger ").append(updateTriggerName(baseTable)).endOfStatement();
+    buffer.append("drop trigger ").append(deleteTriggerName(baseTable)).endOfStatement();
+  }
+
+  @Override
+  protected void createTriggers(DdlWrite writer, MTable table) throws IOException {
+System.out.println("FIXME: HistorySupport is work in Progress for MsSQL");
```

Comment:
indent - maybe comment out all other method that creates/drops triggers as trigger syntax is completely different in MS-SQL